lib/ansible/modules/network/bigswitch/bcf_switch.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Ansible module to manage Big Cloud Fabric (versions 4.1) switches
+# (c) 2017, Ted Elhourani <ted@bigswitch.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: bcf_switch
+author: "Ted (@tedelhourani)"
+short_description: Create and remove a bcf switch.
```

Comment:
One bit of feed back we've recently had fun the community is to put the vendor and/or product name in the `short_description` so a more searchable string will appear in http://docs.ansible.com/ansible/list_of_network_modules.html

So I think bigmon needs adding to this.